### PR TITLE
Chelscontact

### PR DESCRIPTION
**Pull request by camckin10**
Used flexbox to create columns for Contact section. 
Question: Should layout be changed to have colored lines in between columns?

### CODE REVIEW

File: src/components/Event.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ const Event = props => {
     }
   }
 
+  //add code to render to page here?
```
Comment (experimatt) [inline src/components/Event.js]: what is this comment referring to?

**Comment by experimatt** [comment]
@camckin10 couple things:

1. There are merge conflicts with `Contact.js` and `_sections.scss`. You likely need to pull the latest changes in master and then `git merge master` to resolve them.
2. It appears that your editor prefers 4 spaces instead of 2 for indentation, and therefore this PR is changing the entire `_section.scss` file to reflect that. Can you change your editor so it defaults to 2 spaces for indentation instead? That way it won't look like you're changing everything in that scss file.

File: src/styles/_sections.scss [hunk 2]
```diff
@@ -1,202 +1,206 @@
...
-    padding: 3rem 1rem;
-
+    @extend %primary-section;
+    display: flex;
+    align-items: center;
+    padding: 4rem;
     .content-wrap {
-      flex-basis: auto;
-      margin: 2rem 0;
+        flex-basis: 65%;
```
Comment (experimatt) [inline src/styles/_sections.scss]: Use 2 spaces for indentation here instead of 2 please

File: src/styles/_sections.scss [hunk 3]
```diff
@@ -1,202 +1,206 @@
...
-    margin: 1rem 0;
-  }
-
-  .register {
-    width: 100%;
-    padding: 1rem 0;
-
-    .button {
-      font-weight: 600;
+    @extend %secondary-section;
```
Comment (experimatt) [inline src/styles/_sections.scss]: Spacing

File: src/styles/_sections.scss [hunk 4]
```diff
@@ -1,202 +1,206 @@
...
-    h2 {
-      margin-bottom: 1.5rem;
-    }
-  }
-
-  @media (max-width: $laptop) {
-    flex-direction: column;
-    padding: 3rem 1rem;
-
+    @extend %primary-section;
```
Comment (experimatt) [inline src/styles/_sections.scss]: Spacing

**Comment by experimatt** [comment]
The mailchimp signup form doesn't look quite right.

1) I think you can remove `real people should not fill this in and expect good things - do not remove this or risk form bot signups` -- that was originally a comment that came with the mailchimp embedded HTML
2) Once you remove that text, the input field and submit button look a little strange stacked on top of each other. I'm not sure if changing the width of the input, the button or adding spacing between the two will solve it, but I bet you can figure out a way to make it look cleaner.

screen shot 2018-06-08 at 11 44 24 am
screen shot 2018-06-08 at 11 47 35 am
screen shot 2018-06-08 at 11 49 49 am

File: src/components/Event.js (hunk 1 above)
Comment (camckin10) [reply src/components/Event.js]: That was a comment for my brain. My bad!

**Comment by camckin10** [comment]
screen shot 2018-06-13 at 9 10 39 pm

Thanks to Jenessa and Matt, the Contact section is ready for PR review again! I just pushed my latest commits to my branch; I worked on the padding and extended the length of the borders for the columns.

**Comment by experimatt** [comment]
Something is still not right about this PR -- if master was merged in correctly, it wouldn't show all of my commits for the community section, or changes  to 20 files.

I'll pull this down locally and take a peek at it to see if I can sort it out.

File: src/components/Event.js (hunk 1 above)
Comment (experimatt) [reply src/components/Event.js]: I'll remove it

**Pull request merged**